defaults/main.yml

```diff
@@ -9,108 +9,221 @@
 
-# .. envvar:: roundcube__git_repo
+# .. envvar:: roundcube__git_repo [[[
 #
 # Roundcube source repository
 roundcube__git_repo: 'https://github.com/roundcube/roundcubemail.git'
 
-
-# .. envvar:: roundcube__git_dest
+                                                                   # ]]]
+# .. envvar:: roundcube__git_dest [[[
 #
 # Roundcube source directory on the host
 roundcube__git_dest: '{{ roundcube__src + "/" + roundcube__git_repo.split("://")[1] }}'
 
-
-# .. envvar:: roundcube__git_tag
+                                                                   # ]]]
+# .. envvar:: roundcube__git_tag [[[
 #
 # Roundcube release tag to deploy
-roundcube__git_version: '1.1.9'
+roundcube__git_version: '1.3.0'
 
-
-# .. envvar:: roundcube__git_checkout
+                                                                   # ]]]
+# .. envvar:: roundcube__git_checkout [[[
 #
 # Default path where Roundcube source files will be deployed
 roundcube__git_checkout: '{{ roundcube__www + "/sites/" +
                              (roundcube__domain if roundcube__domain is string else roundcube__domain[0]) +
                              "/public" }}'
 
+                                                                   # ]]]
+# .. envvar:: roundcube__src [[[
+#
+# Base path for git bare repository with Roundcube source
+roundcube__src: '{{ (ansible_local.root.src
+                      if (ansible_local|d() and ansible_local.root|d() and
+                          ansible_local.root.src|d())
+                      else "/usr/local/src") + "/" + roundcube__user }}'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__www [[[
+#
+# Base web root directory for Roundcube website
+roundcube__www: '{{ (ansible_local.nginx.www
+                    if (ansible_local|d() and
+                        ansible_local.nginx|d())
+                    else "/srv/www" ) + "/" + roundcube__user }}'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__webserver_user [[[
+#
+# Roundcube webserver user (needs read-only access to the website code)
+roundcube__webserver_user: '{{ ansible_local.nginx.user
+                              if (ansible_local|d() and
+                                  ansible_local.nginx|d() and
+                                  ansible_local.nginx.user|d())
+                              else "www-data" }}'
 
+                                                                   # ]]]
+                                                                   # ]]]
 # .. Database configuration [[[1
 #
 # --------------------------
 #   Database configuration
 # --------------------------
 
-# .. envvar:: roundcube__database_user
+# .. envvar:: roundcube__database [[[
 #
-# Roundcube database user account
-roundcube__database_user: 'roundcube'
-
+# Database definition to use from the :envvar:`roundcube__database_map`.
+roundcube__database: 'sqlite-default'
```

Comment:
Not necessarily in this PR, but role could check if a MariaDB database is installed on the host via Ansible facts and switch to MariaDB database automatically - one less thing to handle via inventory. You can see how it's done in `debops.gitlab` role.